build.gradle

```diff
@@ -250,17 +212,33 @@ publishing {
     }
   }
   repositories {
-    // Credentials shall be stored in ~/.gradle/gradle.properties
-    mavenRepos.each { repo ->
-      maven {
-        name repo.name
-        url repo.url
+    maven {
+      name 'project'
+      url "$buildDir/maven-repo"
+    }
+
+    maven {
+      name 'eclipse'
+      url 'https://repo.eclipse.org/content/repositories/golo-' + (version.endsWith('-SNAPSHOT') ? 'snapshots/' : 'releases/')
+      if (project.hasProperty('eclipseRepoUsername')) {
         credentials {
```

Comment:
What do you mean by "issue"? You have two repositories, so you get two publishing tasks. This makes it easy to discover what the build can do by just running `gradle tasks`. You should always strive to make the build's capabilities understandable without having to read through the whole build script.

If you want to publish to all repos, there is always the `publish` task.